test/functional/api/vim_spec.lua

```diff
@@ -1036,5 +1036,13 @@ describe('api', function()
       eq(expected, nvim("list_uis"))
     end)
   end)
+  describe('nvim_get_keyboard_mods_state', function()
+    it('give empty dictionary for testing as they are off in CI', function()
+      -- For this test, if you are testing on your PC, turn the capslock,
+      -- Numlock and Scrollock off, this tests the presense of these mods
+      -- and on the environment of testing they are supposed to be turn off
+      eq({}, meths.get_keyboard_mods_state())
```

Comment:
Should always return keys. If the state is unknown return `false` as the value.

Reply:
Oh thats it, I did reverse when I was implementing nvim_get_commands, if they were not there don't add them, BTW I will add it